engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -23,12 +26,12 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
   private val LoggingConfigMountPath = "/logging-config"
   private val ModelConfigMountPath = "/model-config"
 
-  def prepare(processVersion: ProcessVersion, resourcesToMount: MountableResources, determinedReplicasCount: Int): Deployment = {
+  def prepare(processVersion: ProcessVersion, canonicalProcess: CanonicalProcess, resourcesToMount: MountableResources, determinedReplicasCount: Int): Deployment = {
```

Comment:
it would be easier to test if we will got everywhere only typespecificdata instead of full scenario

Reply:
done